account_bank_statement_import_camt/models/account_journal.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2013-2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    camt_import_batch = fields.Boolean()
```

Comment:
Can we name this "Aggregate batch transactions (CAMT)"  and translate it to "Samenvoegen batch transacties (CAMT)" in Dutch?